build/fuses/fuses.json5

```
@@ -5,5 +5,7 @@
   "run_as_node": "1",
   "cookie_encryption": "0",
   "node_options": "1",
-  "node_cli_inspect": "1"
+  "node_cli_inspect": "1",
+  "embedded_asar_signature_validation": "0",
+  "non_asar_app_loading": "1"
```

can we invert the sense of this so it's not phrased as a negative?
```suggestion
  "only_load_app_from_asar": "0"
```

---

I know what you're gonna say, but these are (a) internal only and (b) get auto prefixed with `IsXEnabled` in the generated cpp file.

i.e. this becomes `IsNonAsarAppLoadingEnabled()` with your change it would be `IsOnlyLoadAppFromAsarEnabled()` 🤔

---

Aren't these names exposed to the fuse tool too?

I don't mind a slightly awkward generated function name, but a user will presumably type / read the fuse name at some point.

---

The fuse name is whatever we want to call it in fuses (that enum in `@electron/fuses` is not automatically generated), so this string has no impact on that 😄

---

I still think we're better off matching the names between the fuse tool and the source code, it makes it much easier to grep and read!